src/components/ChatEntry.js

```diff
@@ -1,22 +1,43 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+import { useState } from 'react';
 
 const ChatEntry = (props) => {
+  const [likeButton, setLikeButton] = useState('🤍');
+  const toggleClicked = () => {
+    if (likeButton === '🤍') {
+      setLikeButton('❤️');
+      props.updateLikes(true);
+    } else {
+      setLikeButton('🤍');
+      props.updateLikes(false);
+    }
+  };
```

Comment:
This is ok, however it is essentially creating a second function that calls the function you passed in. While this isn't inherently wrong, it becomes a bit redundant. When we pass functions in as props from one component to the next, we are trying to cut down on the need for making extra functions within our child components. What we could have done is create a variable that is dictated by the like prop and simply called the updateLikes within the button itself.